cedar-lean-cli/tests/integration_tests.rs

```diff
@@ -0,0 +1,932 @@
+use std::path::{Path, PathBuf};
+use std::process::{Command, Output};
+
+fn check_output<P: AsRef<Path>>(output: Output, expected_output_file: P, should_error: bool) {
+    let cli_output = std::str::from_utf8(&output.stdout)
+        .expect("Failed to convert output to string")
+        .to_string();
+    let expected_output = std::fs::read_to_string(expected_output_file.as_ref())
+        .expect("Failed to read expected output file");
+
+    assert!(
+        should_error != output.status.success(),
+        "CLI exited with unexpected error code: {}",
+        output.status.code().unwrap()
+    );
+    assert_eq!(
+        cli_output.trim(),
+        expected_output.trim(),
+        "CLI output does not match expected output"
+    )
+}
+
+/***************************************** Tests for Analysis *****************************************/
+#[test]
+fn test_analyze_policies_tabular_view_box_p1() {
+    let base_path = PathBuf::from("examples/analyze/view_box");
+
+    let output = Command::new("cedar-lean-cli")
+        .arg("analyze")
+        .arg("policies")
+        .arg(base_path.join("policies1.cedar"))
+        .arg(base_path.join("policies.cedarschema"))
+        .output()
+        .expect("Failed to run cedar-lean-cli");
```

Comment:
You could have a helper to build these commands so you only pass `policies1.cedar` and `policies.cedarschema` to it here, for example.